react/components/SearchBar/SearchBar.tsx

```diff
@@ -252,13 +252,10 @@ function SearchBar({
             openMenu,
           }) => (
             <div
-              className={classNames(
-                'relative-m w-100',
-                withModifiers('searchBarInnerContainer', [
```

Comment:
That's really weird, `classNames` was transforming the array in a single string `opened filled` making it an invalid modifier and not applying them.

Reply:
:thonking: